lib/stripe/object_types.rb

```diff
@@ -0,0 +1,90 @@
+# frozen_string_literal: true
+
+module Stripe
+  module ObjectTypes
+    def self.object_types_to_classes # rubocop:disable Metrics/MethodLength
```

Comment:
Nit: maybe rename this to `object_names_to_classes`?

Reply:
+1.

Reply:
👍